packages/lib/modules/pool/actions/LiquidityActionHelpers.ts

```diff
@@ -277,6 +278,17 @@ export function requiresProportionalInput(pool: Pool): boolean {
   return isGyro(pool.type) || isCowAmmPool(pool.type)
 }
 
+// Some pool types do not support AddLiquidityKind.Proportional in the SDK
+export function supportsProportionalAddLiquidityKind(pool: Pool): boolean {
+  if (
+    isV2Pool(pool) &&
```

Comment:
I prefer to add this explicit clause guard instead of a single return because we will have more edge cases (maybe YAGNI but it's ok)